### DISCUSSION TITLE
Create a list of all steps with their description

### DISCUSSION THREAD

**Discussion by soredake** [topgrade-rs/topgrade#764] [Ideas]
Currently there are MANY steps and absolutely no information about them:
[possible values: am, app_man, asdf, atom, audit, bin, bob, brew_cask, brew_formula, bun, bun_packages, cargo, chezmoi, chocolatey, choosenim, composer, conda, config_update, containers, custom_commands, deb_get, deno, distrobox, dkp_pacman, dotnet, emacs, firmware, flatpak, flutter, fossil, gcloud, gem, ghcup, github_cli_extensions, git_repos, gnome_shell_extensions, go, guix, haxelib, helm, home_manager, jetpack, julia, juliaup, kakoune, helix, krew, lure, macports, mamba, miktex, mas, maza, micro, myrepos, nix, node, opam, pacdef, pacstall, pearl, pip3, pip_review, pip_review_local, pipupgrade, pipx, pkg, pkgin, pnpm, powershell, protonup, raco, rcm, remotes, restarts, rtcl, ruby_gems, rustup, scoop, sdkman, self_update, sheldon, shell, snap, sparkle, spicetify, stack, stew, system, tldr, tlmgr, tmux, toolbx, vagrant, vcpkg, vim, vscode, winget, wsl, wsl_update, yadm, yarn]

On what system this steps can be run, what are they doing, like, what config_update doing? Or restarts?

So, it would be nice to have a list of them, with description like where they can be run and what are they doing.

**Comment by SteveLauC** [comment]
see the discussion in #756

**Comment by GideonBear** [comment]
Replaced by #761
